docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -40,6 +40,56 @@ Let's take a look at a sample recipe document:
         "sodium": 959.0,
     }
 
+Sample queries with HTTP client
```

Comment:
Not sure why we moved it to the start, I think we need to load data first before being able to make requests